migrations/2023-07-17-111427_add-fraud-check-table.sql/up.sql

```diff
@@ -0,0 +1,34 @@
+-- Your SQL goes here-- Your SQL goes here
+CREATE TYPE "FraudCheckType" AS ENUM (
+    'pre_frm',
+    'post_frm'
+);
+
+CREATE TYPE "FraudCheckStatus" AS ENUM (
+    'fraud',
+    'manual_review',
+    'pending',
+    'legit',
+    'transaction_failure'
+);
+
+CREATE TABLE fraud_check (
+    id SERIAL PRIMARY KEY,
```

Comment:
remove id field

Reply:
why remove 'id'?
we previously discussed that every table should have an id field
later, if in the worst case, we do certain migrations manually, the id field will be of much help

Reply:
We need to `move away from auto increment ids`  for any distributed storage related approaches. And migration without downtime with auto-increment id's also hard. So don't need this field